.circleci/config.yml

```diff
@@ -1,154 +1,5 @@
 version: 2
```

Comment:
Any reason why this is still here?
if we deactivate circle.. this will be useless anywauy

Reply:
I removed it on another PR... if we merge this one, and then the other one, we will remove circleci completely.